if-else questions/absolute.cpp

```diff
@@ -0,0 +1,20 @@
+#include <iostream>
+
+using namespace std;
+
+int main()
+{
+    int n;
+    cin >> n;
+    if (n > 0)
+    {
```

Comment:
no need of writing `std::` again and again. We have already declared using namespace std;